robot_ws/src/object_tracker_robot/scripts/run_turtlebot_controller.sh

```diff
@@ -2,5 +2,9 @@
 
 set -ex
 
```

Comment:
Thanks to ROS2 launch system, `run_turtlebot_controller.sh` is not needed any more. Just figure out the paths inside `main.launch.py` and add one launch action to download the model and another one to start up the inference worker.

Also, you can pass as a parameter the topic name to listen to, so it will change accordingly when using raspicam vs. usbcam.

Reply:
@AAlon  run_turtlebot_controller.sh can be removed and include in launch file, But the same structure we had followed in simulation_ws  also , so do you mean that changes also has to be reverted as per this comment?

Reply:
As suggested, Removed run_turtlebot_controller.sh and created related changes in inference_worker and launch for robot_ws. 

Please find the [PR#41](https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/pull/41) for details.

Reply:
Thank you.